vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -318,6 +324,7 @@ void Contribution::OnBalanceForReconcile(
   auto const contribution =
       braveledger_bind_util::FromStringToContributionQueue(contribution_queue);
   if (result != ledger::Result::LEDGER_OK || !info) {
+    queue_in_progress_ = false;
```

Comment:
Just checking - can this be called on a thread _other_ than the one running 'ProcessContributionQueue'?

Reply:
nope